### DISCUSSION TITLE
Coinbase - DOGE

### DISCUSSION THREAD

**Discussion by zet2002** [taoteh1221/Slideshow_Crypto_Ticker#18] [General]
Hi all!

I recently loaded this to my raspberry pi, running great but hitting a snag.   

When setting:
`exchange_markets['coinbase'] = 'DOGE-USD';`
and then reloading, it will load but it seems to be suck saying "Loading..." and never pulls up the information.  

Is my syntax wrong?  I tried to look up documentation around proper syntax to no avail.

Thanks in advance, and thank you for this project!

**Comment by taoteh1221** [comment]
Hello. Your welcome, glad you are enjoying the ticker! :)

Be sure to check that the market exists on the chosen exchange. I don't believe DOGE is listed on Coinbase. Try Binance, as 'dogeusdt'.

Here are all the markets available on Coinbase: https://api.pro.coinbase.com/products

**Reply by zet2002** [reply]
Thank you so much for that!  That page is what I was looking for as well.

**Reply by taoteh1221** [reply]
No prob, glad you got it figured out.
